functions/exercise2/script.js

```diff
@@ -7,4 +7,17 @@ function isPalindrome(phrase) {
     }
     var reversedNormalizedPhrase = normalizedPhrase.split("").reverse().join("");
     return reversedNormalizedPhrase === normalizedPhrase;
+}
+
+function showResult() {
+    var phrase = document.getElementById("phrase").value;
+    try {
+        var result = isPalindrome(phrase);
+        if (result) {result = "is a palindrome";}
```

Comment:
Please use a new variable to store a text that should be displayed is phrase is a palindrome.